ansible/site.yml

```diff
@@ -125,14 +127,37 @@
   gather_facts: false
   become: True
   tasks:
+    - debug:
+        msg: "Multicloud HA is being installed, skipping Multicloud local install"
+      when: gelato_ha == true
     - import_role:
         name: common
       when: common_role_done is not defined
     - import_role:
         name: gelato-installer
       when:
         - deploy_project != "hotpot"
+        - gelato_ha == false  
   tags: gelato
+ 
+- name: Install Gelato - HA Mode
```

Comment:
Insall gelato HA mode needs to be part of gelato installer role. You can create an issue now and handled later.

Reply:
yes, will optimize depending on the direction .. If other services are to be made HA .. gelato and others will be consumed by gelato-HA  / SODA-HA role..